plugins/api-docs/src/components/GraphQlDefinitionWidget/GraphQlDefinition.tsx

```diff
@@ -38,6 +39,12 @@ const useStyles = makeStyles<BackstageTheme>(() => ({
       },
     },
   },
+  hints: {
```

Comment:
hmm you probably want to use `<Alert` from material-ui at info level instead of hand-winging this, since that gives consistent styling across the app